.github/ISSUE_TEMPLATE/installation_issue.md

```diff
@@ -10,8 +10,8 @@ assignees: ''
 #### Preliminaries
 
 - [ ] I have followed the latest version of the
-      [installation instructions](https://docs.manim.community/en/latest/installation.html).
-- [ ] I have checked the [troubleshooting page](https://docs.manim.community/en/latest/installation/troubleshooting.html) and my problem is either not mentioned there,
+      [installation instructions](https://docs.manim.community/en/stable/installation.html).
+- [ ] I have checked the [troubleshooting page](https://docs.manim.community/en/stable/installation/troubleshooting.html) and my problem is either not mentioned there,
```

Comment:
I wouldn't change it here, since this isn't really linking to documentation that a user would follow. I think a warning/note should be hinted to somewhere else in the docs though about ensuring the documentation they're following is for the version they're using.

Reply:
I have to disagree here; the installation issue template would be used by a user who's having trouble installing manim, and who will most likely want to install the stable version. Linking to the latest version here would be confusing.

Reply:
Alrighty, you've convinced me... but I would still like to see a warning/note somewhere in the installation about making sure the documentation they're looking at matches the version they're as well as a note about how they can choose to install the latest if they'd like to

Reply:
I don't think that's within the scope of this PR.